src/syncerd/bitcoin_syncer.rs

```diff
@@ -86,21 +85,22 @@ fn create_electrum_client(
     electrum_server: &str,
     proxy_address: Option<String>,
 ) -> Result<Client, electrum_client::Error> {
+    let config = ConfigBuilder::new().retry(0);
```

Comment:
for documentation, can you describe why it's necessary/preferable for the electrum client not to attempt reconnection?